usaspending_api/disaster/v2/views/federal_account/loans.py

```diff
@@ -1,38 +1,103 @@
+from typing import List
+
 from django.db.models import F
-from rest_framework.response import Response
 from usaspending_api.accounts.models import TreasuryAppropriationAccount
 from usaspending_api.common.cache_decorator import cache_response
 from usaspending_api.disaster.v2.views.disaster_base import (
-    DisasterBase,
     LoansPaginationMixin,
     LoansMixin,
     FabaOutlayMixin,
 )
-from usaspending_api.disaster.v2.views.federal_account.spending import construct_response
+from usaspending_api.disaster.v2.views.elasticsearch_account_base import ElasticsearchAccountDisasterBase
 
 
-class LoansViewSet(LoansMixin, LoansPaginationMixin, FabaOutlayMixin, DisasterBase):
+class LoansViewSet(LoansMixin, LoansPaginationMixin, FabaOutlayMixin, ElasticsearchAccountDisasterBase):
     """ Returns loan disaster spending by federal account. """
 
     endpoint_doc = "usaspending_api/api_contracts/contracts/v2/disaster/federal_account/loans.md"
+    agg_key = "financial_accounts_by_award.treasury_account_id"  # "financial_accounts_by_award.federal_account_id"  primary (tier-1) aggregation key
```

Comment:
Is this correct? Why is this part here (looks like commented-out previous impl):

> `# "financial_accounts_by_award.federal_account_id"`

I would think that the top-tier agg is the fed account, and then `sub_agg_key` should be `treasury_account_id`

Reply:
Originally yes, but the implementation was changed for performance reasons to use the lower-tier as the agg key and get the toptier data, then combine the results.

Reply:
Ok. probably should remove the  `"financial_accounts_by_award.federal_account_id"` part from the comment then in both places.